tooling/api/src/helpers/event.ts

```diff
@@ -17,12 +17,33 @@ export interface Event<T> {
   payload: T
 }
 
-export type EventName = string
+export type EventName = TauriEvent | string
 
 export type EventCallback<T> = (event: Event<T>) => void
 
 export type UnlistenFn = () => void
 
+export enum TauriEvent {
+  WINDOW_RESIZED = 'tauri://resize',
+  WINDOW_MOVED = 'tauri://move',
+  WINDOW_CLOSE_REQUESTED = 'tauri://close-requested',
+  WINDOW_CREATED = 'tauri://window-created',
+  WINDOW_DESTROYED = 'tauri://destroyed',
+  WINDOW_FOCUS = 'tauri://focus',
+  WINDOW_BLUR = 'tauri://blur',
+  WINDOW_SCALE_FACTOR_CHANGED = 'tauri://scale-change',
+  WINDOW_THEME_CHANGED = 'tauri://theme-changed',
+  WINDOW_FILE_DROP = 'tauri://file-drop',
+  WINDOW_FILE_DROP_HOVER = 'tauri://file-drop-hover',
+  WINDOW_FILE_DROP_CANCELLED = 'tauri://file-drop-cancelled',
+  MENU = 'tauri://menu',
+  CHECK_UPDATE = 'tauri://update',
```

Comment:
```suggestion
  UPDATE = 'tauri://update',
```

Reply:
Addressed in [c7cb454](https://github.com/tauri-apps/tauri/pull/5100/commits/c7cb4545b4828dbd497c165e9ddb3bb01773cdd1).